src/Specs.as

```diff
@@ -117,10 +117,10 @@ public class Specs {
 		["turn @turnLeft %n degrees",			" ", 1, "turnLeft:",				15],
 		["--"],
 		["point in direction %d.direction",		" ", 1, "heading:",					90],
-		["point towards %m.spriteOrMouse",		" ", 1, "pointTowards:",			""],
+		["point towards %m.spriteOrMouse",		" ", 1, "pointTowards:",			"mouse-pointer"],
 		["--"],
 		["go to x:%n y:%n",						" ", 1, "gotoX:y:"],
-		["go to %m.location",				" ", 1, "gotoSpriteOrMouse:",		"mouse-pointer"],
+		["go to %m.location",					" ", 1, "gotoSpriteOrMouse:",		"mouse-pointer"],
```

Comment:
This change makes more sense when tab = 4 spaces.
Try adding `?ts=4` to the end of this page's URL...